src/decorator-annotator.ts

```diff
@@ -57,7 +57,19 @@ class ClassRewriter extends Rewriter {
     if (decSym.flags & ts.SymbolFlags.Alias) {
       decSym = this.typeChecker.getAliasedSymbol(decSym);
     }
-    return decSym.getDocumentationComment().some(c => c.text.indexOf('@Annotation') >= 0);
+    for (let d of decSym.getDeclarations()) {
+      // yuk, getLeadingCommentRangesOfNode is a private API, using it until
+      // https://github.com/Microsoft/TypeScript/issues/7393 is resolved.
+      // Switch to the TS JSDoc parser in the future to avoid false positives here.
```

Comment:
Good call. I didn't realize that some of the utilities.ts functions were exposed.

I decided against extracting into a comment tsickle helper to return JSDoc text, because it is just two calls into the TS apis and I hope we can switch to using the TS JSDoc parser eventually. WDYT?